src/Workspaces/Core/Portable/Recommendations/AbstractRecommendationService.cs

```diff
@@ -132,6 +140,15 @@ internal bool ShouldIncludeSymbol(ISymbol symbol)
                 }
             }
 
+            // Primary constructor parameters should not appear in static context
+            if (_context.IsStatementContext &&
```

Comment:
i don't understaand the IsStatementContext check.

Reply:
It's indeed unnecessary, removed